index.html

```diff
@@ -4,21 +4,33 @@
     <meta charset="UTF-8">
```

Comment:
PR should have a descriptive summary of the features implemented.

Reply:
I have added a comment for the pull request, I hope that can suffice. Thank you